force-app/main/default/aura/TaskCardKanban/TaskCardKanbanController.js

```diff
@@ -0,0 +1,22 @@
+({
+	newTask : function(component, event, helper) {
+		let openNewTaskModal = component.getEvent("openNewTaskCard");
+        openNewTaskModal.setParam("typeOfTask",event.getSource().get("v.name"));
+        openNewTaskModal.fire();
+	},
+    completeTask : function(component, event, helper)
+    {
+        let closeTask = component.getEvent("closeTask");
```

Comment:
you should move this logic into a helper. The controller should be something more like an "event handler"